index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
keeping this at the root level supports both the webpack dev server (and live reloading) and github pages